script/download-node.js

```diff
@@ -110,7 +115,8 @@ var downloadNode = function(version, done) {
   }
 };
 
-downloadNode('v0.10.35', function(error) {
+var apmNodeVersion = process.env.APM_NODE_VERSION || '0.10.35';
```

Comment:
@kevinsawicki I might be wrong. Things are a bit like a black box for me when it comes to Electron and its bundled Node, and the interaction with apm and its Node.